### ISSUE TITLE
Body font is 12px when using the scheme Mist on mobile phone

### ISSUE THREAD

**Issue by zhaoyibo** [theme-next/hexo-theme-next#245]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
Keep the same font size

### Actual behavior 
The text is 12px font, but the code is 14px font

### Steps to reproduce the behavior 
1. use scheme Mist
2. use mobile phone to browse

* Link to demo site with this issue: N/A
* Link(s) to source code or any usefull link(s): N/A

### Node.js and NPM Information

```

```

### Package dependencies Information

```

```

#### Hexo version

```

```

#### Hexo Configuration

```yml

```

### NexT Information

**NexT Version:**

- [ ] Latest Master branch
- [x] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [ ] All schemes
- [ ] Muse
- [x] Mist
- [ ] Pisces
- [ ] Gemini

#### NexT Configuration:

```yml

```

### Other Information 
![](https://ws2.sinaimg.cn/large/006tNc79ly1fqi8ul1s37j3162108tfu.jpg)

This problem only applies to scheme Mist, and there are no problems with the other schemes.

**Comment by maple3142** [comment]
Set `font` option in `_config.yml` manually?
```yaml
font:
  enable: false

  # Uri of fonts host. E.g. //fonts.googleapis.com (Default).
  host:

  # Font options:
  # `external: true` will load this font family from `host` above.
  # `family: Times New Roman`. Without any quotes.
  # `size: xx`. Use `px` as unit.

  # Global font settings used for all elements in <body>.
  global:
    external: true
    family: Lato
    size:

  # Font settings for Headlines (H1, H2, H3, H4, H5, H6).
  # Fallback to `global` font settings.
  headings:
    external: true
    family:
    size:

  # Font settings for posts.
  # Fallback to `global` font settings.
  posts:
    external: true
    family:

  # Font settings for Logo.
  # Fallback to `global` font settings.
  logo:
    external: true
    family:
    size:

  # Font settings for <code> and code blocks.
  codes:
    external: true
    family:
    size:
```

**Comment by zhaoyibo** [comment]
I just switched it from off to on and changed fonts CDN. This is my configuration.
```
font:
  enable: true

  # Uri of fonts host. E.g. //fonts.googleapis.com (Default).
  host: //fonts.loli.net

  # Font options:
  # `external: true` will load this font family from `host` above.
  # `family: Times New Roman`. Without any quotes.
  # `size: xx`. Use `px` as unit.

  # Global font settings used for all elements in <body>.
  global:
    external: true
    family: Lato
    size:

  # Font settings for Headlines (H1, H2, H3, H4, H5, H6).
  # Fallback to `global` font settings.
  headings:
    external: true
    family:
    size:

  # Font settings for posts.
  # Fallback to `global` font settings.
  posts:
    external: true
    family:

  # Font settings for Logo.
  # Fallback to `global` font settings.
  logo:
    external: true
    family: 
    size:

  # Font settings for <code> and code blocks.
  codes:
    external: true
    family:
    size:
```

**Comment by maple3142** [comment]
You can set `size` for each type of text.

**Comment by zhaoyibo** [comment]
It does not work for me. I set the global font size to 14px, but it still show 12px.

**Comment by ivan-nginx** [comment]
> I set the global font size to 14px, but it still show 12px.

So? Where is u configuration? In config abouve font sizes are not setted.

**Comment by zhaoyibo** [comment]
@ivan-nginx Thanks for the reply.
The configuration above was not actually setted, but I tried to set it up, like below, with no effect.
```yml
font:
  enable: true

  # Uri of fonts host. E.g. //fonts.googleapis.com (Default).
  host: //fonts.loli.net

  # Font options:
  # `external: true` will load this font family from `host` above.
  # `family: Times New Roman`. Without any quotes.
  # `size: xx`. Use `px` as unit.

  # Global font settings used for all elements in <body>.
  global:
    external: true
    family: Lato
    size: 14px

  # Font settings for Headlines (H1, H2, H3, H4, H5, H6).
  # Fallback to `global` font settings.
  headings:
    external: true
    family:
    size:

  # Font settings for posts.
  # Fallback to `global` font settings.
  posts:
    external: true
    family:

  # Font settings for Logo.
  # Fallback to `global` font settings.
  logo:
    external: true
    family: 
    size:

  # Font settings for <code> and code blocks.
  codes:
    external: true
    family:
    size:
```

This problem will only appear in scheme Mist. Other schemes do not have this problem.

**Comment by maple3142** [comment]
You can set `font.codes.size` to 14px

**Comment by zhaoyibo** [comment]
The font size of the code is already 14px.
Anyway, I tried it, the body font is still 12px, the font of the code is 14px.

```
  global:
    external: true
    family: Lato
    size: 14px
  codes:
    external: true
    family:
    size: 14px
```

**Comment by ivan-nginx** [comment]
Fix added #262.
